sdks/typescript/src/apache_beam/coders/coders.ts

```diff
@@ -114,20 +114,31 @@ export interface Coder<T> {
   toProto(pipelineContext: ProtoContext): runnerApi.Coder;
 }
 
-function writeByteCallback(val, buf, pos) {
+function writeByteCallback(
+  val: number,
+  buf: { [x: string]: number },
+  pos: number
+) {
   buf[pos] = val & 0xff;
 }
 
+interface HackedWriter extends Writer {
```

Comment:
Should we export this? Presumably we're going to want it in the caller eventually

Reply:
Yes, done.